### DISCUSSION TITLE
Enable RSS in the blog

### DISCUSSION THREAD

**Discussion by himatech** [spacedriveapp/spacedrive#480] [Feature Request]
This is not related to the app in particular. I suggest enabling RSS for the blog to follow its updates without relying on the social media accounts.

**Comment by HeavySnowJakarta** [comment]
> This is not related to the app in particular. I suggest enabling RSS for the blog to follow its updates without relying on the social media accounts.

Try RSSHub for making RSS feeds of social media source
